static/app/views/performance/trends/changeExplorer.tsx

```diff
@@ -0,0 +1,330 @@
+          trendChangeType={trendChangeType}
+          trendFunction={trendFunction}
+          trendView={trendView}
+          statsData={statsData}
+          isLoading={isLoading}
+          organization={organization}
+          projects={projects}
+          trendParameter={trendParameter}
+          location={location}
+        />
+      </PanelBodyWrapper>
+    </SlideOverPanel>
+  );
+}
+
+function ExplorerBody(props: ExplorerBodyProps) {
+  const {
+    transaction,
+    trendChangeType,
+    trendFunction,
+    trendView,
+    trendParameter,
+    isLoading,
+    location,
+    organization,
+  } = props;
+  const breakpointDate = transaction.breakpoint
+    ? new Date(transaction.breakpoint * 1000).toUTCString().replace('GMT', 'UTC')
+    : '';
+  return (
+    <React.Fragment>
+      <Header transaction={transaction} trendChangeType={trendChangeType} />
+      <ExplorerContainer>
+        <ExplorerContainer flex>
+          <InfoItem
+            float
+            margin
+            label={
+              trendChangeType === TrendChangeType.REGRESSION
+                ? t('Regression Metric')
+                : t('Improvement Metric')
+            }
+            value={trendFunction}
+          />
+          <InfoItem label={t('Start Time')} value={breakpointDate} />
+        </ExplorerContainer>
+      </ExplorerContainer>
+      <GraphPanel data-test-id="pce-graph">
+        <strong>{trendParameter.label + ' (' + trendFunction + ')'}</strong>
+        <ExplorerText color={theme.gray300} margin={'-' + space(3)}>
+          {trendView.statsPeriod
+            ? DEFAULT_RELATIVE_PERIODS[trendView.statsPeriod] ||
+              getTimeString(trendView.statsPeriod)
+            : trendView.start + ' - ' + trendView.end}
+        </ExplorerText>
+        <Chart
+          query={trendView.query}
+          project={trendView.project}
+          environment={trendView.environment}
+          start={trendView.start}
+          end={trendView.end}
+          statsPeriod={trendView.statsPeriod}
+          disableXAxis
+          disableLegend
+          neutralColor
+          {...props}
+        />
+      </GraphPanel>
+      <MetricsChart
+        isLoading={isLoading}
+        location={location}
+        transaction={transaction}
+        trendFunction={trendFunction}
+        trendView={trendView}
+        organization={organization}
+      />
+    </React.Fragment>
+  );
+}
+
+function getTimeString(time: string) {
+  const timeMeasurements = {
+    m: 'minutes',
+    h: 'hours',
+    d: 'days',
+    w: 'weeks',
+  };
+
+  const suffix = time.charAt(time.length - 1);
+  const number = time.slice(0, time.length - 1);
+  const measurement =
+    number === '1'
+      ? timeMeasurements[suffix].slice(0, timeMeasurements[suffix].length - 1)
+      : timeMeasurements[suffix];
+
+  const timestring = number === '1' ? measurement : number + ' ' + measurement;
+  return tct('Last [timestring]', {timestring});
+}
```

Comment:
I think a util for `Last <human readable time>` might already exist. I'd maybe do `grep -rn 'Last' static | grep -i time` and check.